packages/master-detail-layout/theme/lumo/vaadin-master-detail-layout-styles.js

```diff
@@ -0,0 +1,28 @@
+import '@vaadin/vaadin-lumo-styles/color.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-master-detail-layout',
+  css`
+    :host(:is([overlay], [stack])) [part='detail'] {
+      background-color: var(--lumo-base-color);
+    }
+
+    :host([overlay]) [part='detail'] {
+      box-shadow: var(--lumo-box-shadow-s);
+    }
+
+    :host([overlay][orientation='horizontal']) [part='detail'] {
+      border-inline-start: 1px solid var(--lumo-contrast-10pct);
+    }
+
+    :host([overlay][orientation='vertical']) [part='detail'] {
+      border-block-start: 1px solid var(--lumo-contrast-10pct);
+    }
+
+    :host([overlay]) [part='master'] {
```

Comment:
Can we make this more like an overlay, for example using a pseudo element? Also, should we handle viewport containment as well?

Reply:
Right, I missed the viewport containment. I'd probably add dedicated part for it rather than a pseudo element, e.g. `backdrop` part like we have in `vaadin-dialog` and `vaadin-app-layout` for consistency. We could hopefully replace that with `::backdrop` pseudo-element in the future when switching to native popover API.

Reply:
Added the part with corresponding styles and visual tests.